lemur/plugins/lemur_acme/ultradns_record.py

```diff
@@ -0,0 +1,26 @@
+class Record:
```

Comment:
Could this class exist in ultradns.py? Just to keep it all in one file, since the other plugins operate this way